app/i18n/it/sub.php

```diff
@@ -12,78 +12,78 @@
-			'_' => 'Content action when fetching the article content',	// TODO
-			'append' => 'Add after existing content',	// TODO
-			'prepend' => 'Add before existing content',	// TODO
-			'replace' => 'Replace existing content',	// TODO
+			'_' => 'Azione da effettuare quando viene recuperato il contenuto di un articolo',
+			'append' => 'Aggiungi dopo il contenuto esistente',
+			'prepend' => 'Aggiungi prima del contenuto esistente',
+			'replace' => 'Rimpiazza il contenuto esistente',
 		),
-		'css_cookie' => 'Use Cookies when fetching the article content',	// TODO
-		'css_cookie_help' => 'Example: <kbd>foo=bar; gdpr_consent=true; cookie=value</kbd>',	// TODO
+		'css_cookie' => 'Usa i cookie quando viene recuperato il contenuto di un articolo',
+		'css_cookie_help' => 'Esempio: <kbd>foo=bar; gdpr_consent=true; cookie=value</kbd>',
 		'css_help' => 'In caso di RSS feeds troncati (attenzione, richiede molto tempo!)',
 		'css_path' => 'Percorso del foglio di stile CSS del sito di origine',
 		'css_path_filter' => array(
-			'_' => 'CSS selector of the elements to remove',	// TODO
-			'help' => 'A CSS selector may be a list such as: <kbd>.footer, .aside</kbd>',	// TODO
+			'_' => 'Il selettore CSS degli elementi da rimuovere',
+			'help' => 'Il selettore CSS potrebbe essere una lista, ad esempio: <kbd>.footer, .aside</kbd>',
 		),
 		'description' => 'Descrizione',
 		'empty' => 'Questo feed non contiene articoli. Per favore verifica il sito direttamente.',
 		'error' => 'Questo feed ha generato un errore. Per favore verifica se ancora disponibile.',
 		'filteractions' => array(
-			'_' => 'Filter actions',	// TODO
-			'help' => 'Write one search filter per line.',	// TODO
+			'_' => 'Azioni di filtro',
+			'help' => 'Scrivi un filtro di ricerca per riga.',
 		),
 		'information' => 'Informazioni',
 		'keep_min' => 'Numero minimo di articoli da mantenere',
 		'kind' => array(
-			'_' => 'Type of feed source',	// TODO
+			'_' => 'Tipo di sorgente del feed',
 			'html_xpath' => array(
-				'_' => 'HTML + XPath (Web scraping)',	// TODO
+				'_' => 'HTML + XPath (Web scraping)',	// IGNORE
 				'feed_title' => array(
-					'_' => 'feed title',	// TODO
-					'help' => 'Example: <code>//title</code> or a static string: <code>"My custom feed"</code>',	// TODO
+					'_' => 'titolo del feed',
+					'help' => 'Esempio: <code>//titolo</code> o una stringa statica: <code>"Il mio feed personalizzato"</code>',
 				),
-				'help' => '<dfn><a href="https://www.w3.org/TR/xpath-10/" target="_blank">XPath 1.0</a></dfn> is a standard query language for advanced users, and which FreshRSS supports to enable Web scraping.',	// TODO
+				'help' => '<dfn><a href="https://www.w3.org/TR/xpath-10/" target="_blank">XPath 1.0</a></dfn> è un linguaggio di ricerca standard per utenti avanzati supportato da FreshRSS per abilitare il Web scraping.',
 				'item' => array(
 					'_' => 'finding news <strong>items</strong><br /><small>(most important)</small>',	// TODO
```

Comment:
There are a few (20) missing  `// TODO` such as this line, if you feel like completing this file

Reply:
These are the ones I had some problems with, could you please tell me where to find these strings in the demo so I can have a bit of context on how they are used? Thanks

Reply:
See for instance:

![image](https://user-images.githubusercontent.com/1008324/194297036-49deef56-3d5c-47ce-8819-3585c2a67460.png)